cmd/iconbridge/chain/near/sender.go

```diff
@@ -88,34 +88,24 @@ func (s *Sender) Segment(ctx context.Context, msg *chain.Message) (tx chain.Rela
 		Receipts: make([][]byte, 0),
 	}
 
-	var msgSize uint64
+	receipt := msg.Receipts[0]
```

Comment:
I suggest you to configure the size of the btp message transaction from the configuration file. In the `bmr.config.json` file, in the `dst` field, you can provide options. You can set the value to be able to support as many message as near can process. It can be figured out after a few trial and errors.
```JSON
      "dst": {
       ...
        "options": {
          ...
          "tx_data_size_limit": 8192,
          ...
        },
 ```

Reply:
Resolved by https://github.com/icon-project/icon-bridge/pull/545/commits/1915df5fb0d8bdab6d644444c1aa1dc59142f1ac